src/Configuration/ConfigurationFactory.php

```diff
@@ -369,12 +369,9 @@ private function retrieveFilter(string $filter, ?string $gitDiffFilter, bool $is
             return $filter;
         }
 
+        $gitDiffFilter ??= 'AM';
         $baseBranch = $gitDiffBase ?? $this->gitDiffFileProvider->provideDefaultBase();
 
-        if ($isForGitDiffLines) {
-            return $this->gitDiffFileProvider->provide('AM', $baseBranch, $sourceDirectories);
-        }
-
```

Comment:
I find this less clear tbh. in which way it is more explicit?

in the previous version it's immediately clear that if `isForGitDiffLines` is `true`, then `AM` is used. Now it takes time to understand it, isn't it?